internify/src/components/molecules/pages/TotalStudents.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { ChipNote } from "../../atoms/Chips";
+import "./styles/Feedback.css";
+
+const TotalStudents = ({ data }) => {
+  const msg1 = `${data.length} total students`;
+  return (
+    <div className="feedback_page1_inner">
+      <ul className="feedback_list">
+        <li>
+          {" "}
```

Comment:
There's been a lot of these empty strings scattered throughout the past PR's. Please get rid of them in the future.